neps/nep-0246.md

```diff
@@ -0,0 +1,550 @@
+// * `token_ids`: the tokens to transfer 
+// * `amounts`: the number of tokens to transfer, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+//    array with 128 bits.
+interface MtTransferLog {
+    authorized_id?: string,
+    old_owner_id: string,
+    new_owner_id: string,
+    token_ids: string[],
+    amounts: string[],
+    memo?: string
+}
+```
+
+## Examples
+
+Single owner minting (pretty-formatted for readability purposes):
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_mint",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora", "proximitylabs_ft"], "amounts":["1", "100"]}
+  ]
+}
+```
+
+Different owners minting:
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_mint",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora", "proximitylabs_ft"], "amounts":["1","100"]},
+    {"owner_id": "user1.near", "token_ids": ["meme"], "amounts": ["1"]}
+  ]
+}
+```
+
+Different events (separate log entries):
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_burn",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora", "proximitylabs_ft"], "amounts": ["1","100"]},
+  ]
+}
+```
+
+Authorized id:
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_burn",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora_alpha", "proximitylabs_ft"], "amounts": ["1","100"], "authorized_id": "thirdparty.near" },
+  ]
+}
+```
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_transfer",
+  "data": [
+    {"old_owner_id": "user1.near", "new_owner_id": "user2.near", "token_ids": ["meme"], "amounts":["1"], "memo": "have fun!"}
+  ]
+}
+
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_transfer",
+  "data": [
+    {"old_owner_id": "user2.near", "new_owner_id": "user3.near", "token_ids": ["meme"], "amounts":["1"], "authorized_id": "thirdparty.near", "memo": "have fun!"}
+  ]
+}
+```
+
+## Further Event Methods
+
+Note that the example events covered above cover two different kinds of events:
+1. Events that are not specified in the MT Standard (`mt_mint`, `mt_burn`)
+2. An event that is covered in the [Multi Token Core Standard](https://nomicon.io/Standards/MultiToken/Core.html#mt-interface). (`mt_transfer`)
+
+This event standard also applies beyond the three events highlighted here, where future events follow the same convention of as the second type. For instance, if an MT contract uses the [approval management standard](https://nomicon.io/Standards/MultiToken/ApprovalManagement.html), it may emit an event for `mt_approve` if that's deemed as important by the developer community.
+
+Please feel free to open pull requests for extending the events standard detailed here as needs arise.
+
```

Comment:
Please add section [Reference Implementation](https://github.com/near/NEPs/blob/master/nep-0000-template.md#reference-implementation). If you already have a repository implementing the spec above, you can link the minimum viable interface (trait) and implementation in the Reference Implementation section. See [NEP-0171](https://github.com/near/NEPs/blob/master/neps/nep-0171.md#reference-implementation) as an example. Optionally, inlined code is also acceptable.

Reply:
I've added reference implementation links, they're not in the standard as of yet but it should help people figure things out. I linked to the current community implementations master repo. @jlogelin lmk what you think.